repos/system_upgrade/el7toel8/actors/opensshprotocolcheck/tests/unit_test.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.actor import library
+from leapp.models import OpenSshConfig, OpenSshPermitRootLogin, Report
+from leapp.snactor.fixture import current_actor_context
+
+
+def test_no_config(current_actor_context):
+    with pytest.raises(StopActorExecutionError):
+        library.process(iter([]))
+
+
+osprl = OpenSshPermitRootLogin(value='no')
+
+
+@pytest.mark.parametrize('protocol', ['', '1', '2', '1,2', '2,1', '7'])
+def test_protocol(current_actor_context, protocol):
+    if protocol:
```

Comment:
Nit: I suspect L19-27 can be squashed into `current_actor_context.feed(OpenSshConfig(permit_root_login=[osprl], protocol=protocol))` if no_protocol test case is passed as None.

Reply:
Yeah, thanks.